robocat/README.md

```diff
@@ -0,0 +1,153 @@
+# Setting up the Robocat cluster from scratch
+
+These are step by step instructions on how to setup the `robocat` cluster using
+the available automation. The automation is based on Tekton, so it requires
+a cluster with Tekton deployed, which will run the tasks required to setup
+the `robocat` cluster. These "driver" cluster is the `dogfooding` cluster.
+
+# Point your cluster config to `robocat`
+
+The initial step is done on the `robocat` cluster directly, so point your
+configuration to it:
+
+```
+kubectl config use-context gke_tekton-nightly_europe-north1-a_robocat
+```
+
+# Create a Cluster Admin service account
+
+To setup the [cluster admin](root/README.md) service account, authenticate to
+the cluster with an admin user, and apply the content of the `root` folder:
+
+```
+kubectl apply -f robocat/root
+```
+
+Create a secret `robocat-tektoncd-cadmin-token` in the `dogfooding` cluster,
+that holds the token for the `cadmin`. This secret is used by the
+`robocat-cadmin` pipeline resource, which is used by the automation to drive
+deployments in the `robotcat` cluster.
+
+```
+# Fetch the secret data from robocat
+CADMIN_SECRET=$(kubectl get -n tektoncd sa/cadmin -o jsonpath='{.secrets[0].name}')
+CA_CRT=$(kubectl get -n tektoncd secret/$CADMIN_SECRET -o jsonpath='{.data.ca\.crt}')
+TOKEN=$(kubectl get -n tektoncd secret/$CADMIN_SECRET -o jsonpath='{.data.token}')
+
+# Create the secret on dogfooding
+cat <<EOF | kubectl --cluster gke_tekton-releases_us-central1-a_dogfooding create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: robocat-tektoncd-cadmin-token
+type: Opaque
+data:
+  ca.crt: $CA_CRT
+  token: $TOKEN
+EOF
+```
+
+# Verify the robocat cluster resources
+
+Obtain the URL of the cluster:
+
+```
+kubectl cluster-info
+```
+
+Ensure that the [`robocat-tekton-deployer`](https://github.com/tektoncd/plumbing/blob/5f9cb51b8530f9bfc5e97e235980767ae53cdec9/tekton/resources/cd/clusters.yaml#L57)
+and the [`robocat-cadmin`](https://github.com/tektoncd/plumbing/blob/5f9cb51b8530f9bfc5e97e235980767ae53cdec9/tekton/resources/cd/clusters.yaml#L77) resources point to the correct URL of the cluster.
+If not fix them in git and re-apply them to the `dogfooding` cluster.
+
+# Wait... or not
+
+Almost everything else is setup automatically via cronjobs scheduled in the
+`dogfooding` cluster. Since the setup of the DNS entry and the creation of the
+`ClusterIssuer` at the right time are still not automated, it's best for now
+to run through the setup "manually" by triggering the various cronjobs
+one by one, at least for the initial setup.
+Future hanges to the resources will be deployed nightly from git.
+
+# Point your cluster config to `dogfooding`
```

Comment:
My understanding is that this is part of running the CronJobs manually so second level heading  makes more sense 
```suggestion
## Point your cluster config to `dogfooding`
```

Reply:
Done